pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
 packages = [{ include = "data_diff" }]
 
 [tool.poetry.dependencies]
-python = "^3.7.1"
+python = "^3.8.0"
```

Comment:
Do we intentionally drop Python 3.7 while it is still officially maintained (until June 2023)?

Reply:
No, we do not. It just was for testing because updating the lock with python 3.7 was x100 slower than for python 3.8.
New changes reverts python 3.7 back.